p2panda-net-next/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+mod actors;
+mod addrs;
+mod network;
+
+/// Unique 32 byte identifier for a network.
+///
+/// Peers operating on the same network identifier will eventually discover each other. This is the
+/// most global identifier to group peers into networks. Different applications may choose to share
+/// the same underlying network infrastructure by using the same network identifier.
+///
+/// Please note that the network identifier should _never_ be the same as any other topic
+/// identifier.
+pub type NetworkId = [u8; 32];
```

Comment:
Maybe we keep that out for now as this is not used by the PR, I believe we need to revisit now with the refactoring what the role of a "network id" is in general.

Reply:
Good call. I've removed it.